action.yml

```diff
@@ -20,6 +20,7 @@ runs:
     # dependencies, but that doesn't occur when we're running against a raw checkout.
     - run: |
         go-licenses check ./... --ignore $(go list -m) --include_tests \
-        --ignore $(tr -d ' \n' <<<"${{ inputs.ignore-modules }}")
+        --ignore $(tr -d ' \n' <<<"${{ inputs.ignore-modules }}") \
+        --ignore $(go list std | awk 'NR > 1 { printf(",") } { printf("%s",$0) } END { print "" }')
```

Comment:
A comment would have been helpful here but people can `git blame` and check the PR, I guess

Reply:
I added an explanatory comment.